content/toolsAndResources/nge/ngeBlocks.md

```diff
@@ -0,0 +1,1387 @@
+
+#### Properties
+- **Operation** allows this node to be changed to one of the following functions: Cos, Sin, Abs, Exp, Round, Floor, Ceiling, Sqrt, Log, Tan, ArcTan, ArcCos, ArcSin, Sign, Negate, OneMinus, Reciprocal, ToDegrees, or ToRadians.
+
+#### Inputs
+- **input** is connected to a node providing a value to be used in the function.
+
+#### Outputs
+- **output** is the principle square root of the input value in a type that matches the input value type. 
+
+![Subtract node](/img/tools/nge/subtractNode.jpg)
+
+### Subtract
+This is a simple math node which subtracts the right input from the left input. This node can support Float, Int, Vector2, Vector3, and Vector4 types, though the node cannot add mixed types. The type of the node first connected to an input determines the type for the node output. 
+
+#### Properties
+- **Operation** allows this node to be changed to one of the following operations: Add, Subtract, Multiply, Divide, Min, or Max.
+
+#### Inputs
+- **left** is connected to a node providing the first value to be used in the operation.
+- **right** is connected to a node providing the second value to be used in the operation.
+
+#### Outputs
+- **output** is the difference of the right input subtracted from the left input and matches type with the values connected to left and right. 
+
+## Math: Scientific
+These nodes are a collection of more complex mathematical functions, mostly trigonometric. They are all very straight forward nodes with one input and one output. All nodes in this set can support Float, Int, Vector2, Vector3, and Vector4 types.
+
+![Absolute node](/img/tools/nge/absNode.jpg)
+
+### Abs
+This is a simple function to return the absolute value of the input, which is always a positive value. This node can support Float, Int, Vector2, Vector3, and Vector4 types. The type for the node connected to the input of this node determines the type for the node output. 
+
+#### Properties
+- **Operation** allows this node to be changed to one of the following functions: Cos, Sin, Abs, Exp, Round, Floor, Ceiling, Sqrt, Log, Tan, ArcTan, ArcCos, ArcSin, Sign, Negate, OneMinus, Reciprocal, ToDegrees, or ToRadians.
+
+#### Inputs
+- **input** is connected to a node providing a value to be used in the function.
+
+#### Outputs
+- **output** is the absolute value of the input value in a type that matches the input value type. 
+
+![Arc cosine node](/img/tools/nge/arcCosNode.jpg)
+
+### ArcCos
+This function returns the inverse cosine of the input and assumes radians for the input value. This node can support Float, Int, Vector2, Vector3, and Vector4 types. The type for the node connected to the input of this node determines the type for the node output. 

```

Comment:
Actually, the input of the arcXXX functions is unitless, as a cos/sin/tan/... result is a ratio of two numbers that have the same unit.

`cos(x) = y` => y is unitless. So `x=arccos(y)` => the input of `arccos` is unitless.

Reply:
fixed in https://github.com/BabylonJS/Documentation/pull/963/commits/507d9500883016a65d3957fd84105dc8e640a259